helm/credentiald-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: credentiald
+rules:
```

Comment:
Can we specify that `credentiald` should only be able to create secrets in the giantswarm namespace?